### PR TITLE
Backlinked missing ACL workshops

### PR DESCRIPTION
**Pull request by mjpost**

### CODE REVIEW

**Comment by mjpost** [comment]
I'm not sure why the build is breaking. Maybe comments aren't allowed in the XML?

**Review by anthology-assist — APPROVED** [review]
LGTM

File: data/xml/2023.ws.xml [hunk 1]
```diff
@@ -2,43 +2,49 @@
 <collection id="2023.ws">
   <event id="ws-2023">
     <colocated>
-      <volume-id>2023.bsnlp-1</volume-id>
```
[the remark this answers is not in the document]
Comment (anthology-assist) [reply data/xml/2023.ws.xml:5]: Saw the reordering.

**Comment by mjpost** [comment]
Okay, this is nice. @anthology-assist, we can now add comments in the XML files. When adding workshops to `YEAR.ws.xml`, please group them by host event, as seen [here for example](https://github.com/acl-org/acl-anthology/blob/acl-workshops/data/xml/2023.ws.xml).

image

**Comment by mbollmann** [comment]
For the new library, I would like to make it possible to both read and write back Anthology data. Comments in the XML make this significantly harder. Just noting this here because I think it'd have been good to at least briefly discuss this new feature before merging.

**Comment by mbollmann** [comment]
(I do think having comments is a great idea from a usability perspective, though.)

**Comment by mbollmann** [comment]
Oh, and it also means the PyPI package breaks with the new Anthology data, which we should probably start thinking about now that we're starting to advertise it, if mainly to paper reviewers... (I really need to merge it in here ASAP so we can do this properly.)

**Comment by mjpost** [comment]
Hmm, I didn't really think of this as a new feature or consider it would cause other problems. Actually I was surprised that comments are not just included in the tag tails. If this adds too much complexity it's easy enough to revert it. We could achieve the same effect in this situation by adding substructure to the WS backlinks, with a real tag.

**Comment by mbollmann** [comment]
Anything that needs to modify `bin/anthology/*` might need a corresponding change in the PyPI package, or it's not guaranteed that it'll still work. This means we should probably be careful with such changes before we've made the switch to the new package. (And it means I should hurry up with that switch, of course...)

To elaborate on my first concern:

I think it's desirable to use our library not just for parsing, but also for writing Anthology data. That way, changes to the XML format only require changes within the library — as of now, we also need to remember to update ingestion and other scripts that operate on the XML directly. In fact, I've put off several tasks I've wanted to do that require modifying data until I have that functionality in place. But allowing XML comments means we need to store them somehow during parsing, not just discard them, or we can't later write them back. So a proper implementation of comments would be a bit more complex than what's in this PR.

**Comment by mjpost** [comment]
Sounds good. I very much agree about being able to read the XML and write it out exactly as it was, so you're right about this. I'll create another PR to revert the comment piece.

**Pull request merged**